odoo/addons/base/models/res_partner.py

```diff
@@ -432,7 +432,6 @@ def _commercial_sync_to_children(self):
         sync_children = self.child_ids.filtered(lambda c: not c.is_company)
         for child in sync_children:
             child._commercial_sync_to_children()
-        sync_children._compute_commercial_partner()
```

Comment:
this would be a regression, I think
https://github.com/odoo/odoo/commit/0d68acff8e4436e55676a5016e0496b7c3aa35aa

Reply:
You are right, I had missed that fix. I'm gonna do it in a slightly different way ;)

Reply:
Fixed here https://github.com/odoo-dev/odoo/commit/cd5e447383171259964aa9beadf88ce2342ab5a1